interfaces/builtin/basedeclaration.go

```diff
@@ -327,6 +327,8 @@ slots:
       slot-snap-type:
         - core
     deny-auto-connection: true
+  ofono:
+    deny-auto-connection: true
```

Comment:
Now that the base declaration is more finalized (also see https://github.com/snapcore/snapd/pull/2269), please adjust this to be:
```
   ofono:                                                              
     allow-installation:                                                         
       slot-snap-type:                                                           
         - app                                                                   
         - core                                                                  
     deny-auto-connection: true                                                  
     deny-connection:                                                            
       on-classic: false 
```

This says that this may be supplied as an app snap or via core (ie, on classic) and if supplied via an app snap, then a snap declaration is required to connect to the the slot implementation.